ppl/src/main/java/org/opensearch/sql/ppl/parser/AstExpressionBuilder.java

```diff
@@ -253,6 +243,24 @@ public UnresolvedExpression visitConvertedDataType(ConvertedDataTypeContext ctx)
     return AstDSL.stringLiteral(ctx.getText());
   }
 
+  @Override
+  public UnresolvedExpression visitDatetimeConstantLiteral(DatetimeConstantLiteralContext ctx) {
+    return visitFunction(ctx.getText(), null);
+  }
+
+  private Function visitFunction(String functionName, FunctionArgsContext args) {
+    if (args == null) {
```

Comment:
I'd change this to one return statement and use `?:` to construct the `funcArgs` argument.

Reply:
Thanks. Copied from SQL AST builder, so I will update both.